testsuite/analyze_energy.py

```diff
@@ -23,23 +28,29 @@ def setUpClass(self):
         self.system.non_bonded_inter[1, 1].lennard_jones.set_params(
             epsilon=1.0, sigma=1.0,
             cutoff=2**(1. / 6.), shift="auto")
-        self.system.part.add(id=0, pos=[1, 2, 2], type=0)
-        self.system.part.add(id=1, pos=[5, 2, 2], type=0)
+        self.system.part.add(id=0, pos=[1, 2, 2], type=0, q=1)
+        self.system.part.add(id=1, pos=[5, 2, 2], type=0, q=-1)
         self.system.thermostat.set_langevin(kT=0., gamma=1.)
         self.system.bonded_inter.add(self.harmonic)
+        p3m = electrostatics.P3M(bjerrum_length=1.0, accuracy=1e-7)
```

Comment:
Can you please fix some of the p3m parameters to appropriate values, so that the tuning is faster?